### PR TITLE
Added Application XnConvert

### PR DESCRIPTION
**Pull request by SHR1SHAK**
XnConvert is a fast, powerful and free cross-platform batch image converter. Its fast and can handle any type of image file format conversion. 
https://www.xnview.com/en/xnconvert/

### CODE REVIEW

File: config/applications.json [hunk 1]
```diff
@@ -2942,5 +2942,13 @@
 		"description": "Fork - a fast and friendly git client.",
 		"link": "https://git-fork.com/",
 		"winget": "Fork.Fork"
+	},
+	"XnConvert": {
+		"category": "Utilities",
+		"choco": "xnview",
+		"content": "xnconvert.install",
```
Comment (og-mrk) [inline config/applications.json:2949]:
```suggestion
		"content": "XnConvert",
```
@SHR1SHAK I think using the Application Name would be better.

**Comment by SHR1SHAK** [comment]
Thanks for the suggestion @og-mrk.

**Comment by SHR1SHAK** [comment]
I have also fixed the choco repo link as I got confused and inserted another of xn product but its fixed.

**Review by og-mrk — APPROVED** [review]
Cool app addition @SHR1SHAK 👍

**Comment by SHR1SHAK** [comment]
Thanks you @og-mrk

**Review by Real-MullaC — APPROVED** [review]

**Comment by ChrisTitusTech** [comment]
Not accepting any more programs at this time. We will be moving most of the "niche" programs into a button utilizing UniGetUI.

**Pull request closed without merging**